evalme/text/text.py

```diff
@@ -160,7 +160,7 @@ def spans_iou(self, x, y):
             return 0
 
         # in case when substrings are presented - when can compute IOU using them
-        if x.get('text') is not None and y.get('text') is not None:
+        if x.get('text') and y.get('text'):
```

Comment:
I think we should completely avoid text-based IOU computation if-branch. It's quite strange to rely on texts here - it could be a case when two identical texts have different xpaths, and we're failing here.

Reply:
Fixed